textproc/lilypond-devel/Portfile

```diff
@@ -21,59 +22,150 @@ long_description    Lilypond is a unix-based automated engraving system that \
 homepage            http://lilypond.org/
 conflicts           lilypond
 platforms           darwin
-master_sites        http://download.linuxaudio.org/lilypond/sources/v${branch}/
+master_sites        http://lilypond.org/download/sources/v${branch}/
+# master_sites      http://download.linuxaudio.org/lilypond/sources/v${branch}/
 distname            ${my_name}-${version}
-checksums           rmd160  f0d923cec535cf8a133f2c801082db6120f01ad6 \
-                    sha256  3679db30a3f74ce8668691952fb0dd58d8ad6bad099017e815ca3e4312261453
+checksums           rmd160 71f199afa24a75cfde53337c2140e96c74d7857d \
+                    sha256 a5679cddb3e415828642f75027997596b814922aadb1fa633c8f973238ae291b \
+                    size   17407520
 
 depends_build       port:t1utils \
                     port:bison \
                     port:texi2html \
                     port:netpbm \
                     port:pkgconfig \
-                    port:flex
-depends_lib         port:fontconfig \
+                    port:flex \
                     port:fontforge \
+                    port:libtool
+
+depends_lib         port:fontconfig \
                     port:freetype \
                     port:gettext \
                     path:lib/pkgconfig/glib-2.0.pc:glib2 \
                     port:ghostscript \
                     port:gmp \
-                    port:mftrace \
                     port:guile18 \
                     port:texinfo \
                     path:lib/pkgconfig/pango.pc:pango \
-                    port:flex \
-                    port:t1utils \
                     port:texlive \
                     port:texlive-fonts-recommended \
                     port:texlive-lang-cyrillic \
                     port:texlive-metapost \
                     port:dblatex \
-                    port:libtool \
-                    port:python27
+                    port:python27 \
+                    port:urw-core35-fonts
 
+configure.cmd       autoconf -f && ./configure
```

Comment:
It might make sense to use `use_autoconf yes` instead (I need to check the exact expression, I'm not behind my usual toolchain right now) and then change the command for `autoconf` only, rather than artificially squeezing this into `configure` command.

Reply:
Well, the package comes with a `configure` script, which means that `configure` is run twice, since `use_autoconf` is handled rather late.  Additionally, because upstream lilypond expects the availability of `libguile.h` while macports provides `libguile18.h`, it would be necessary to artificially extend the 'post-patch' phase to modify `configure` before it gets regenerated.  This is ugly...

I've checked various Portfiles, and my solution isn't unheard of, so to say :-)